dev_cycle/terraform/input.tf

```diff
@@ -6,3 +6,14 @@ variable "key_pair_name" {}
 variable "machine_name" {}
 variable "vpc_id" {}
 variable "aws_iam_role_id" {}
+
+variable "install" {
+  type = object({
+    linuxkit_download_url = string
+    babashka_download_url = string
```

Comment:
Can we use git tags and/or SHAs and default to 'master' if none is provided.

Reply:
updated